test/util.spec.js

```diff
@@ -244,4 +245,25 @@ describe('util', () => {
       ).to.deep.equal(output);
     });
   });
+
+  describe('fixPathSeparators', () => {
+    it('Should transform any forwardslash into backslash and remove any leftover backslash if OS is Windows', () => {
+      let output;
+
+      const input =
+        '\\Users\\mattlewis\\Code\\/open-source/karma-coverage-istanbul-reporter\\test\\fixtures\\typescript';
+
+      const isWin = process.platform.startsWith('win');
```

Comment:
Can you change this to:
```
Object.defineProperty(process, 'platform', {
        value: 'win32',
      });
```

and then you can remove the `isWin` check below so the test will always run regardless of if you're on windows or not. Other than that, LGTM! 😄

Reply:
Done, I divided this Unit test into two, one for Windows and other for Linux, thanks!